CMakeLists.txt

```diff
@@ -1587,8 +1587,20 @@ if(HEX_FILES_TO_MERGE)
   message(VERBOSE "Merging hex files: ${HEX_FILES_TO_MERGE}")
 endif()
 
-if(EMU_PLATFORM)
-  include(${ZEPHYR_BASE}/cmake/emu/${EMU_PLATFORM}.cmake)
+if(EMU_PLATFORM OR SUPPORTED_EMU_PLATFORMS)
+  if(NOT EMU_PLATFORM)
+    list(GET SUPPORTED_EMU_PLATFORMS 0 EMU_PLATFORM)
```

Comment:
In twisterlib.py, the default value of simulation is "na", https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/pylib/twister/twisterlib.py#L1525. And if cmake choose a default value from `SUPPORTED_EMU_PLATFORMS`, the cmake should give a message to tell user which `EMU_PLATFORM` it choose

Reply:
Hi, there was a warning message here initially, but I've got a [review comment](https://github.com/zephyrproject-rtos/zephyr/pull/38102#discussion_r699219226) that it should be removed.